generic_expression_patterns_modules/download_recount2_data.R

```diff
@@ -14,10 +14,10 @@ get_recount2_template_experiment <- function(project_id,
                                              raw_template_filename) {
 
   # Save current working directory
-  original_wd = getwd()
+  #original_wd = getwd()
```

Comment:
As I mentioned in #29, if you comment out these `getwd()` and `setwd()` lines, you have to call `download_study()` later like this:
```
project_dir <- paste(download_dir, project_id, sep='/')
download_study(project_id, outdir=project_dir)
```
otherwise the downloaded files would be saved in current working directory, whatever it is.

The first test case is not affected because it didn't process the downloaded template file. Instead, it uses the raw template file that already exists in the repo. 
Also note that changing this function will affect `human_general_analysis/1_process_recount2_data.ipynb`.

Reply:
You can try to read the downloaded template file right after the downloading step and see whether it is located in `template_download_dir`. I bet it is not.

Reply:
That makes sense. Thank you for checking. I have updated the script